### PR TITLE
[AWS SSM] install the agent

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Add "Install the Agent with AWS SSM"

### Motivation
Trello request

### Preview link
https://docs-staging.datadoghq.com/ruth/aws-ssm/agent/faq/install-the-agent-with-aws-ssm/

### Additional Notes
I verified this using AWS demo account

### CODE REVIEW

File: content/en/agent/faq/install-the-agent-with-aws-ssm.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager.
+
+### EC2 instance
+* Create an [EC2 IAM Role][1] with the `AmazonEC2RoleforSSM` prebuilt policy or update your current [AWS Datadog policy][2].
+* Launch an [EC2 instance][3] with this role attached.
+
+### Parameter Store
+In the [Parameter Store][4], create a parameter with:
+
+* Name: `dd-api-key-for-ssm`
```
Comment (aalbertson) [inline content/en/agent/faq/install-the-agent-with-aws-ssm.md]: Can be more generic/standard name, I chose that for identification while I was testing.

File: content/en/agent/faq/install-the-agent-with-aws-ssm.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+title: Install the Agent with AWS SSM
+kind: faq
+further_reading:
+- link: "agent/"
+  tag: "Documentation"
+  text: "Learn more about the Datadog Agent"
+---
+
+Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager.
```
Comment (aalbertson) [inline content/en/agent/faq/install-the-agent-with-aws-ssm.md]: Might add a note about this being an "example configuration for a single manual instance being deployed". Beyond launching the instance, the rest of the steps `should` apply to autoscaling infrastructure, etc...

File: content/en/agent/faq/install-the-agent-with-aws-ssm.md (hunk 1 above)
Comment (ruthnaebeck) [reply content/en/agent/faq/install-the-agent-with-aws-ssm.md]: Actually yours had your name in it. I changed it to this here and in the JSON. This way, the person following the instructions doesn't have to update the name in the JSON.

File: content/en/agent/faq/install-the-agent-with-aws-ssm.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: Install the Agent with AWS SSM
+kind: faq
+further_reading:
+- link: "agent/"
+  tag: "Documentation"
+  text: "Learn more about the Datadog Agent"
+---
+
+Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager. **Note**: The EC2 step is for a single manual instance. The other steps can be used for multiple instances.
```
Comment (apigirl) [inline content/en/agent/faq/install-the-agent-with-aws-ssm.md]:
```suggestion
Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager. 
**Note**: The EC2 step is for a single manual instance. Use the other steps for multiple instances.
```

File: content/en/agent/faq/install-the-agent-with-aws-ssm.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+Under [Run Command][7], click the **Run command** button and follow the steps below:
+
+* **Command document**:
+    * Click on the search box and select *Owner -> Owned by me*.
+    * Click the radio button next to your document.
+    * If necessary, choose the **Document version**.
+* **Targets**:
+    * Select the EC2 instance to target.
+* **Output options** (optional):
+    * Select the **CloudWatch output** check box to log any issues.
```
Comment (apigirl) [inline content/en/agent/faq/install-the-agent-with-aws-ssm.md]:
```suggestion
    * Select the **CloudWatch output** checkbox to log any issues.
```

**Comment by ruthnaebeck** [comment]
@aalbertson - Is this good to merge?

**Comment by aalbertson** [comment]
lgtm

**Pull request merged**